pkg/stockings/stock.go

```diff
@@ -0,0 +1,159 @@
+package stockings
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+/*
+	Could look to make this an interface as we move forward with different stock apis
+	ie make different structs for td, iex, rh apis and implement each of these methods for the
+	different apis via StockAPI interface.
+
+*/
+type Stock struct {
```

Comment:
Could we try following the bdc_v4 model structure? [stock](https://github.com/jaysunmah/bdc_v4/blob/master/backend/bluedresscapital/models.py#L14-L16) and [stock quote](https://github.com/jaysunmah/bdc_v4/blob/master/backend/bluedresscapital/models.py#L51-L54) - trying to combine stock meta data with stock prices is a huge headache to maintain, best to just separate them

Reply:
Sure so just to be clear, stock struct just holds ticker and name, stock quote will hold a stock struct and price and date?